core/schema/sdk.go

```diff
@@ -633,6 +633,34 @@ func (sdk *goSDK) baseWithCodegen(
 				},
 			},
 		},
+	}
+
+	// inject sdk specific env variables before withExec
+	// for now allow only Env with prefix GO
+	cfg, ok, _ := src.Self.ModuleConfig(ctx)
```

Comment:
@rajatjindal @jedevc  IIUC this env thing will only work for the Go SDK right? did we create an issue somewhere so the other SDKs could potentially implement this as well?